src/bot/events/verification.js

```diff
@@ -1,10 +1,23 @@
 import {EventListener} from 'yuuko';
 import createLogger from 'another-logger';
 const log = createLogger({label: 'event:verification'});
+import config from '../../../config';
+import {formatDate, formatDateRelative} from '../util/discord';
 
 // Check if members are already verified when being added to a guild
 // TODO more verification hardcoding
-export default new EventListener('guildMemberAdd', async (guild, member, {db}) => {
+export default new EventListener('guildMemberAdd', async (guild, member, {db, client}) => {
+	if (member.bot) return;
+	if (guild.id !== config.TEMP_guildID) return;
+
+	client.createMessage(config.TEMP_joiningChannelID, {
+		content: `User **<@${member.id}> (${member.username}#${member.discriminator})** has joined 
+		\nAccount Created:  ${formatDate(new Date(member.createdAt))} (${formatDateRelative(new Date(member.createdAt))})`,
```

Comment:
Are you sure this has the same format as the screenshot you posted? Newlines and whitespace in backtick string literals are taken literally, so this will end up sending a message like `...has joined(\n\t\t\t\n)Account created...` which isn't *really* a problem but I tend to be picky about not sending extra whitespace like that in message bodies.

Probably just keep this all on one line for now even though it's ugly, or break it out into its own function that builds the content in a variable rather than stuffing the whole thing in a single string expression

Reply:
Yeah, sorry, I fixed that